src/main/java/com/klin/holoItems/collections/gamers/okayuCollection/items/MoguMogu.java

```diff
@@ -0,0 +1,211 @@
+        // "B"lock of netherite
+        recipe.setIngredient('b', new RecipeChoice.MaterialChoice(Material.NETHERITE_BLOCK));
+        // "H"elmet (netherite)
+        recipe.setIngredient('h', new RecipeChoice.MaterialChoice(Material.NETHERITE_HELMET));
+
+        Bukkit.getServer().addRecipe(recipe);
+    }
+
+    @Override
+    public void ability(FoodLevelChangeEvent event){
+        autoEat(event);
+    }
+
+    /**
+     *
+     * @param event - The FoodLevelChangeEvent
+     * @return The ItemStack used to refill the food level.
+     */
+    public static ItemStack autoEat(FoodLevelChangeEvent event) {
+        if(event.getFoodLevel() > 19){
+            // Each drumstick is 2, and each half-drumstick is 1
+            return null;
+        }
+
+        HumanEntity humanEntity = event.getEntity();
+        Player player = Bukkit.getPlayer(humanEntity.getUniqueId());
+        PlayerInventory inv = humanEntity.getInventory();
+        List<Integer> slots = Arrays.asList(0,1,2,3,4,5,6,7,8,40);
+        Collections.shuffle(slots);
+        for(int slot : slots){
+            ItemStack itemStack = inv.getItem(slot);
+            if(itemStack == null)
+                continue;
+
+            // Next, check if we can eat it and if we want to eat it.
+            Item holoItem = Utility.findItem(itemStack, Item.class);
+
+            if(holoItem != null) {
+                // HoloItem!
+                // The HoloItem can handle its own shit.
+                if (holoItem instanceof Consumable) {
+                    // Well now I don't know what to do.
+                    // The holoItem requires a playerItemConsumeEvent
+                    // and playerItemConsumeEvent requires a player, an ItemStack, and an EquipmentSlot.
+                    // The problem is that if it's in a Shulker box it's not in an EquipmentSlot.
+                    if(player != null) {
+                        if(!tryConsumeItem(player, itemStack)){
+                            // do not consume the item
+                            continue;
+                        }
+                    }
+                }
+                // else: holoItem not instance of consumable
+                // holoitem can be "consumable" like spacebread
+                // I don't know how
+                continue;
+            }
+
+            // Not a HoloItem. Run the normal items check.
+            // First check if it's even possible:
+            if(!tryConsumeItem(player, itemStack)) {
+                // do not consume the item
+                // could be an FF item
+                continue;
+            }
+
+            // Then check if it's edible:
+            Material foodType = itemStack.getType();
+            if (!foodType.isEdible()) {
+                // Item isn't edible, so don't consider eating it.
+                continue;
+            }
+
+            // We're eating this item.
```

Comment:
After this point, it should only be the call to finishUsingItem. everything gets handled there by vanilla mechanics